src/contexts/transaction.context.tsx

```diff
@@ -0,0 +1,37 @@
+import { createContext, FC, useContext, useMemo, useState } from "react";
+
+import { chains } from "src/constants";
+import tokens from "src/assets/tokens/tokens.json";
+import { Transaction } from "src/domain";
+
+interface TransactionContext {
+  transaction: Transaction;
+  setTransaction: (transaction: Transaction) => void;
+}
+
+const transactionContextDefaultValue: TransactionContext = {
+  transaction: { chainFrom: chains[0], chainTo: chains[1], token: tokens[0], amount: 0 },
```

Comment:
I'm seeing that we initiate the values instead of be `undefined`
![Screenshot_20220411_111427](https://user-images.githubusercontent.com/19190248/162704899-accf7aeb-a213-4f01-8316-6911a5cb5e02.png)
![Screenshot_20220411_111542](https://user-images.githubusercontent.com/19190248/162705635-3c7f53a1-1e55-43a3-80d6-384e07619d1d.png)

Reply:
Only when the value is not arbitrary. For example, we have a clear default for `PREFERRED_CURRENCY` and both the `snackbar` and the `modal` should be closed by default.

Reply:
In this case is not arbitrary as well, the first step should be from eth to hez. I could set it as constant instead set it in context

Reply:
I think that there is a problem here. @adrifdez is using the data in this context as the data source of the form.

The right approach should be to keep the data in this context to `undefined` and let the `TransactionForm` handle its data source with the default values. When the user submits the `form` we need to set the transaction data in this context.

Reply:
Yep, this makes more sense.